metrictank-sample.ini

```diff
@@ -19,9 +19,11 @@ metric-max-stale = 3h
 # Interval to run garbage collection job
 gc-interval = 1h
 
-# duration before secondary nodes start serving requests
-# shorter warmup means metrictank will need to query cassandra more if it doesn't have requested data yet.
-# in clusters, best to assure the primary has saved all the data that a newly warmup instance will need to query, to prevent gaps in charts
+# duration before non-primary (secondary or query) nodes start serving requests
+# shorter warmup on secondary nodes means metrictank will need to query cassandra more if it doesn't have requested data yet.
+# in clusters, best to assure the primary has saved all the data that a newly warmed up instance will need to query,
```

Comment:
this sentence doesnt make sense.  Perhaps
```
To prevent gaps in charts when running a cluster of nodes you need to either have the nodes backfill data from Kafka (set via "offset" in the "kafka-mdm-in" config) or set the warm-up-period to a value long enough to ensure all data received before the node started has been persisted to the backend store.
```

Reply:
It does makes sense, but yours is much more specific, and thus better.